cmd/snapweb/handlers.go

```diff
@@ -107,6 +107,21 @@ func handleTimeInfo(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func handleBranding(h *snappy.Handler, w http.ResponseWriter, r *http.Request) {
+	brandingData, err := snappy.GetBrandingData(h)
+	if err != nil {
+		log.Println("Cannot get branding data", err)
+		// brandingData is still valid and we fallback to defaults
```

Comment:
Can you test a unit test to verify that the default branding is indeed returned in case we of errors?